routes/web.php

```diff
@@ -46,8 +50,8 @@
         Route::name('update')->patch('/', 'UsersController@update');
         Route::name('bookmarks.show')->get('/bookmarks', 'UsersController@showBookmarks');
         Route::name('delegations.show')->get('/delegations', 'UserDelegationsController@show');
-        Route::name('confirm-destroy')->get('/confirm-destroy', 'UsersController@confirmDestroy');
-        Route::name('destroy')->delete('/', 'UsersController@destroy');
+        Route::get('confirm-delete', [UsersController::class, 'confirmDelete'])->name('confirm-delete');
+        Route::delete('delete', [UsersController::class, 'destroy'])->name('delete');
```

Comment:
Call the `name()` method first for consistent style with the other route declarations.